client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -193,19 +194,39 @@ const SubscriberDataViews = ( {
 			},
 			{
 				id: 'plan',
-				label: translate( 'Subscription type' ),
+				label: translate( 'Type' ),
 				getValue: ( { item }: { item: Subscriber } ) =>
 					item.plans?.length ? SubscribersFilterBy.Paid : SubscribersFilterBy.Free,
 				render: ( { item }: { item: Subscriber } ) => <SubscriptionTypeCell subscriber={ item } />,
 				elements: [
-					{ label: 'Paid', value: SubscribersFilterBy.Paid },
-					{ label: 'Free', value: SubscribersFilterBy.Free },
+					{ label: translate( 'Paid' ), value: SubscribersFilterBy.Paid },
+					{ label: translate( 'Free' ), value: SubscribersFilterBy.Free },
 				],
 				filterBy: {
 					operators: [ 'is' as Operator ],
 				},
+				enableHiding: false,
 				enableSorting: true,
+			},
+			{
+				id: 'is_email_subscriber',
+				label: translate( 'Email subscriber' ),
+				getValue: ( { item }: { item: Subscriber } ) => ( item.is_email_subscriber ? 'yes' : 'no' ),
+				render: ( { item }: { item: Subscriber } ) => (
+					<div>{ item.is_email_subscriber ? 'Yes' : 'No' }</div>
+				),
+				elements: [
+					{ label: translate( 'Subscribed' ), value: SubscribersFilterBy.EmailSubscriber },
+					{
+						label: translate( 'Not subscribed' ),
+						value: SubscribersFilterBy.ReaderSubscriber,
```

Comment:
The back-end is expecting `reader_subscriber` for the non-email-subscribed filter.